modules/ROOT/pages/java-support.adoc

```diff
@@ -199,3 +212,4 @@ For more information, see xref:dataweave::dataweave-formats-java.adoc#java-suppo
 == See Also
 
 * https://help.salesforce.com/s/articleView?id=000396936&type=1[MuleSoft Java 17 Upgrade FAQ^]
+* xref:release-notes::connector/anypoint-connector-release-notes.adoc[Connector Release Notes]
```

Comment:
Since this isn't specific to all of Java 17 and is already mentioned in the doc, maybe we can remove it

Reply:
@sathya0 I think it's okay to leave it. In case someone knows which connector they want to check, that page will link them to their specific connector.